troubleshooting/tools/invalid-utf-8/invalid.py

```diff
@@ -0,0 +1,33 @@
+# small script that writes byte values to a file
+# which if interpreted as utf-8 will be invalid chars
+# can test FLB processing of invalid unicode
+
+myfile = open('log.log', 'ab')
+
+# from https://github.com/fluent/fluent-bit/pull/4297
+invalid = b'\xEF\xBF\x00'
+
+# these were copied from invalid utf-8 examples online
```

Comment:
should we add source link ?

Reply:
like the line 7

Reply:
added: https://stackoverflow.com/questions/1301402/example-invalid-utf8-string